src/client/src/ui/blotter/model/blotterModel.js

```diff
@@ -64,8 +64,8 @@ export default class BlotterModel extends ModelBase {
           (tradesUpdate:TradesUpdate) => {
             let trades = tradesUpdate.trades;
 
-            if (tradesUpdate.isStateOfTheWorld){
-              this.trades = this.trades.concat(tradesUpdate.trades);
+            if (tradesUpdate.isStateOfTheWorld) {
+              this.trades = this.trades.concat(_.sortBy(tradesUpdate.trades, 'tradeId').reverse());
```

Comment:
Is it possible that if multiple batches of trades are  received the final sort order might be incorrect?